docs/usage/self-hosting.md

```diff
@@ -254,13 +254,13 @@ Note: the GitHub.com token in env is necessary in order to retrieve Release Note
 
 You should save and test out this script manually first, and add it to cron once you've verified it.
 
-## Kubernetes for Gitlab, using Git over SSH
+## Kubernetes for GitLab, using Git over SSH
```

Comment:
Question, does changing capitalization here break any link back to this section?

Reply:
I don't think so, but now 100 % sure

Reply:
I think it's ok